### DISCUSSION TITLE
What is the JSVM v23 equivalent of echo.Context.file?

### DISCUSSION THREAD

**Discussion by benallfree** [pocketbase/pocketbase#5759] [Q&A] [answered]
Trying to find the new way to respond with content from file https://pocketbase.io/jsvm/interfaces/echo.Context.html#file

**Comment by ganigeorgiev** [comment, marked as answer]
There is no direct equivalent because we don't have `c.echo.Filesystem` and I'm not sure that we need it to be honest. This is an old API (_I think it predates the Go `fs.FS` interface_) and it does the same thing as calling [`fileFS`](https://pocketbase.io/jsvm-rc/interfaces/core.RequestEvent.html#fileFS), the only difference is that you need to be explicit about the `fs.FS` argument.

For example to serve a file from the local disk, you should be able to do something like:

```js
routerAdd("GET", "/example", (e) => {
    return e.fileFS($os.dirFS("/path/to/your/dir"), "example.txt")
})
```
